src/methods/format-date.js

```diff
@@ -35,7 +35,9 @@ const formatDate = (date, lang = config.get('lang'), options) => {
  * @returns {string}
  *
  * @example
- * // TODO
+ * const notifications = { datetime: '2019-06-19T03:35:52.811Z', content: {api_event: {resource: 'orders'}}}
```

Comment:
It's just one notification, so not notificationS:

```suggestion
 * const notification = { datetime: '2019-06-19T03:35:52.811Z', content: { api_event: { resource: 'orders' } } }
```